src/Public/js/myradio.timeslot.js

```diff
@@ -1,3 +1,14 @@
+//Format names function for concating names;
```

Comment:
not sure the comment is really necessary, and it reads quite confusingly to me

Reply:
`//combines the first,nickname,last name of the user together`
would this be good or show I remove the comment fully

Reply:
probably best to just yeet it